### PR TITLE
implement wildcard support for `skip_tags`

### LINKED ISSUE #9: allow skip-tags with wildcard
A feature I'm missing from this action is the ability to skip multiple tags by matching with a wildcard

This would be for example:

```
- uses: snok/container-retention-policy@v1
  with:
    image-names: dev, web, test
    cut-off: two hours ago UTC+2
    timestamp-to-use: updated_at
    account-type: org
    org-name: google
    keep-at-least: 1
    skip-tags: latest, latest-*
    token: ${{ secrets.PAT }}
```

This would allow me to not cleanup tags which match latest-*

It would be even better if there was regex support, but for that to work, I don't think the `,` separator would work well, thinking of `\d{0,2}` for example

Maybe if it was instead/also accepted

```
skip-tags: | 
  latest
  v.*
```

### PR DESCRIPTION
**Pull request by Emilgardis**
uses fnmatch for Unix shell-style wildcards

resolves #9

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -185,9 +186,13 @@ async def get_and_delete_old_versions(image_name: ImageName, inputs: Inputs, htt
...
-        if any(tag in inputs.skip_tags for tag in image_tags):
-            # Skipping because this image version is tagged with a protected tag
-            continue
+        skip = False
+        for skip_tag in inputs.skip_tags:
+            if any(fnmatch(tag, skip_tag) for tag in image_tags):
+                # Skipping because this image version is tagged with a protected tag
+                skip = True
+        if skip:
+            break
```
Comment (sondrelg) [inline main.py:195]: Why break instead of continue?
```suggestion
            continue
```

**Review by sondrelg — COMMENTED** [review]
Thanks for opening a PR. Just have one question before looking over once more.

File: main.py (hunk 1 above)
Comment (Emilgardis) [reply main.py:195]: no reason except for I did it wrong :D

Comment (Emilgardis) [reply main.py]: fixed this and pushed

**Comment by sondrelg** [comment]
Looks good 👍 Thanks!

**Pull request merged**